### PR TITLE
Create Skills/Bots Page: Make Preview Consistent with Standard Chat Box

### LINKED ISSUE #2751: Create Skills/Bots Page: Make Preview Consistent with Standard Chat Box
For the preview window on https://susi.ai/skillWizard and https://susi.ai/botWizard please reuse the standard chat box of the system.

Screenshot from 2019-07-31 20-17-48

### PR DESCRIPTION
**Pull request by harsh-jindal**
Fixes #2751 
Demo Link: https://pr-2773-fossasia-susi-web-chat.surge.sh

Screenshots of the change: [Add screenshots depicting the changes.]
Screenshot 2019-08-04 at 8 13 36 AM

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@akshatnitd Made the changes. I didn't get the part of separate message store. Do you mean use the `fetchConversationResponse` api along with a message state as was used earlier ?

**Comment by harsh-jindal** [comment]
@akshatnitd Please review.

**Comment by harsh-jindal** [comment]
@fragm3 I understood what @akshatnitd  meant here but the issues says to reuse the chatapp component. Also the messages are displayed in the same way as the current one. Shall I just make the UI of preview chat same without using the chatapp component ?

**Comment by fragm3** [comment]
@harsh-jindal yes, I think that would be better since the response varies and we don't require the functionalities of the main chat app. This PR implementation makes the chat app difficult to understand and we should follow separation of concern here. So, just copy the styling and UI. Thanks!

File: src/components/cms/BotBuilder/Preview/Preview.js [hunk 1]
```diff
@@ -327,16 +332,7 @@ class Preview extends Component {
   render() {
     const { messages, previewChat, message } = this.state;
     const {
-      design: {
-        botbuilderBackgroundBody,
```
Comment (akshatnitd) [inline src/components/cms/BotBuilder/Preview/Preview.js]: We still want this functionality to design the Bot Preview.

**Comment by harsh-jindal** [comment]
@akshatnitd Done.

**Comment by harsh-jindal** [comment]
@fragm3 Removing box shadow isn't a good idea. I guess. Maybe decrease the shadow would be better.
I'll make the other changes.
Screenshot 2019-08-04 at 11 42 09 AM

**Comment by mariobehling** [comment]
Does It work with voice input? At today's meeting we saw that it is important to consider.

**Comment by harsh-jindal** [comment]
@mariobehling No, as suggested by @akshatnitd and @fragm3 to keep the implementation of chatapp and preview different as they need to have different message stores. This PR fixes the UI part only.

**Comment by harsh-jindal** [comment]
@fragm3 Done.
Screenshot 2019-08-04 at 4 21 47 PM

**Comment by mariobehling** [comment]
We need to work more on this. The left side bar spacing does not look anymore as expected when expanded for example. On small screens the chat icon does not show etc.

Screenshot from 2019-08-05 10-53-53


Screenshot from 2019-08-05 10-52-55

**Pull request merged**